pkg/coscheduling/core/core.go

```diff
@@ -269,13 +269,14 @@ func (pgMgr *PodGroupManager) calculateAssignedPods(podGroupName, namespace stri
 	return count
 }
 
-func (pgMgr *PodGroupManager) CheckClusterResource(nodeList []*framework.NodeInfo, resourceRequest corev1.ResourceList) error {
+// CheckClusterResource check the cluster left resources, will return resource gap if left resources can not satisfy the request
```

Comment:
```suggestion
// CheckClusterResource checks if resource capacity of the cluster can satisfy <resourceRequest>.
// It returns an error detailing the resource gap if not satisfied; otherwise returns nil.
```